yarn-project/acir-simulator/src/avm/opcodes/decode_bytecode.ts

```diff
@@ -17,17 +17,18 @@ export function decodeBytecode(bytecode: Buffer): Instruction[] {
     const opcodeByte = bytecode[bytePtr];
     bytePtr += AVM_OPCODE_BYTE_LENGTH;
     if (!(opcodeByte in Opcode)) {
-      throw new Error(`Opcode ${opcodeByte} not implemented`);
+      throw new Error(`Opcode 0x${opcodeByte.toString(16)} not implemented`);
     }
     const opcode = opcodeByte as Opcode;
 
     const instructionType = INSTRUCTION_SET.get(opcode);
     if (instructionType === undefined) {
-      throw new Error(`Opcode ${opcode} not implemented`);
+      throw new Error(`Opcode 0x${opcode.toString(16)} not implemented`);
```

Comment:
Instruction Set in YP shows hex opcodes, so thought this would be a good change.

Reply:
SG